web/src/beta/components/fields/SliderField/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Property from "@reearth/beta/components/fields";
+import Slider, { Props as ToggleProps } from "@reearth/beta/components/Slider";
+
+export type Props = {
+  name?: string;
+  description?: string;
+} & ToggleProps;
```

Comment:
why don't  we use a general props like adding type file and import the shared props from it , instead of mentioned another field props

Reply:
You mean for the name and description?